velocity/tool/src/templates/VM_chef_library.vm

```diff
@@ -542,7 +542,7 @@ ${extra}")
 *# ------------------------------------------------------------------
 #macro (chef_setupformattedtextarea $textarea_id)
 	#if (!$is_wireless_device)
-		<script type="text/javascript" defer="1">sakai.editor.launch('$textarea_id');</script>	
```

Comment:
is this change needed?

Reply:
Yes. Defer does nothing in this case as it only works on actual sources script loads. This is just an inline script which calls a function.